src/libraries/System.Text.RegularExpressions/tests/RegexParserTests.cs

```diff
@@ -42,10 +34,10 @@ static RegexParserTests()
         [InlineData(@"[a\x00(?#)b]", RegexOptions.None, null)]
         [InlineData(@"[a\u0000(?#)b]", RegexOptions.None, null)]
         [InlineData(@"[a\](?#)b]", RegexOptions.None, null)]
-        [InlineData("(?", RegexOptions.None, RegexParseError.UnrecognizedGrouping)]
-        [InlineData("(?", RegexOptions.IgnorePatternWhitespace, RegexParseError.UnrecognizedGrouping)]
-        [InlineData("(? ", RegexOptions.None, RegexParseError.UnrecognizedGrouping)]
-        [InlineData("(? ", RegexOptions.IgnorePatternWhitespace, RegexParseError.UnrecognizedGrouping)]
+        [InlineData("(?", RegexOptions.None, RegexParseError.InvalidGroupingConstruct)]
```

Comment:
Can you please update the tests to verify the offset?

Reply:
This isn't so trivial, unfortunately (mainly: there are a lot of tests...), but I'm working on it.

Reply:
@danmosemsft, I've added 120 or so tests for offset verification (well, not really added, I modified existing tests to test for offset, after all, the data was already there).

I think that was the last open item from the reviews.